cabal.project

```diff
@@ -11,9 +11,9 @@ repository cardano-haskell-packages
 
 index-state:
   -- Bump both the following dates if you need newer packages from Hackage
-  , hackage.haskell.org 2024-09-12T00:00:00Z
+  , hackage.haskell.org 2024-09-10T13:49:28Z
   -- Bump this if you need newer packages from CHaP
-  , cardano-haskell-packages 2024-09-12T00:00:00Z
```

Comment:
@zeme-wana I wasn't sure whether or not these timestamps were automatically set, but it didn't work for me. It led ti

```
Error: [Cabal-7159]
Latest known index-state for 'cardano-haskell-packages' (...) is older than the requested index-state (...).
```

Hence the update.